src/app/process-editor/components/process-modeler/palette/palette-overlay.directive.ts

```diff
@@ -0,0 +1,70 @@
+ /*!
+ * @license
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Directive, ElementRef, TemplateRef, AfterViewInit, Input, HostListener, ViewContainerRef } from '@angular/core';
+import { OverlayRef, PositionStrategy, Overlay } from '@angular/cdk/overlay';
+import { TemplatePortal } from '@angular/cdk/portal';
+
+ @Directive({
+     selector: '[ama-palette-overlay]'
+ })
+
+ export class PaletteOverlayDirective implements AfterViewInit {
+    overlayPosition: PositionStrategy;
+    templatePortal: TemplatePortal;
+    @Input() templatePortalContent: TemplateRef<any>;
+    @Input() amaPaletteItem: any;
+    @Input() amaPaletePortal: TemplatePortal;
```

Comment:
Do you use this Input at all? there is a typo, and because of it I'm not sure it is used.